src/shared/services/email/email.service.ts

```diff
@@ -20,35 +16,47 @@ const transporter = createTransport({
   },
 });
 
+const getStringFile = async (): Promise<string> => {
+  const filePath = path.resolve(process.cwd(), 'html/email-index.html');
+
+  const stringFile = await new Promise<string>((res) => {
+    fs.readFile(filePath, async (err, data) => {
```

Comment:
Change callback from `fs` to promises from `fs/promises`:
`import fsp from 'fs/promises'`